modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -100,13 +100,24 @@ The following Premium plugin has been announced as reaching its end-of-life:
 [[accompanying-enhanced-skins-and-icon-packs-changes]]
 == Accompanying Enhanced Skins & Icon Packs changes
 
-The {productname} <x.y[.z]> release includes an accompanying release of the **Enhanced Skins & Icon Packs**.
+The {productname} 7.0 release includes an accompanying release of the **Enhanced Skins & Icon Packs**.
 
 === Enhanced Skins & Icon Packs
 
 The **Enhanced Skins & Icon Packs** release includes the following updates:
 
-The **Enhanced Skins & Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} <x.y[.z]> skin, Oxide.
+The **Enhanced Skins & Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 7.0 skin, Oxide.
+
+==== Fixed some focus issues related to {productname} 5 skin and premium skins
```

Comment:
Do you reckon mentioning TinyMCE 5 in the heading could add potential confusion? Could we just omit `5` from the heading?

Reply:
```suggestion
==== Fixed focus issues related to Enhanced Skins and Icon Packs
```

Reply:
Isn't this change relevant to all skins though?

Reply:
Yeah, 😢

Reply:
I've updated the suggestion, unless anyone else has a better way of explaining it?